doc/api/cli.md

```diff
@@ -2294,6 +2294,16 @@ The following environment variables are set when running a script with `--run`:
 * `NODE_RUN_PACKAGE_JSON_PATH`: The path to the `package.json` that is being
   processed.
 
+### `--run-dir=<dir>`
+
```

Comment:
```suggestion

> Stability: 1.0 - Early development

```

And should we add `experimental` to the flag name?

Reply:
I saw that we added to the others